template-parts/header_content-title_subtitle.php

```diff
@@ -0,0 +1,33 @@
+<?php
+$obj           = ucfwp_get_queried_object();
+$title         = ucfwp_get_header_title( $obj );
+$subtitle      = ucfwp_get_header_subtitle( $obj );
+$h1            = ucfwp_get_header_h1_option( $obj );
+$h1_elem       = ucfwp_get_header_h1_elem( $obj );
+$title_elem    = ( $h1 === 'title' ) ? $h1_elem : 'span';
+$subtitle_elem = ( $h1 === 'subtitle' ) ? $h1_elem : 'span';
+
+$clusters_link = get_theme_mod( 'clusters_list_page' ) ? get_permalink( get_theme_mod( 'clusters_list_page' ) ) : null;
+?>
+
+<?php if ( $title ): ?>
+	<div class="header-content-inner align-self-start pt-4 pt-sm-0 align-self-sm-center">
+		<div class="container">
+			<?php if ( $clusters_link ) : ?>
+			<div class="d-inline-block bg-inverse-t-2 p-3 mb-1">
+				<a class="text-inverse text-uppercase letter-spacing-3 font-weight-light" href="<?php echo $clusters_link; ?>"><span class="fa fa-chevron-left"></span> UCF Faculty Clusters</a>
```

Comment:
Remove the space between the icon `span` and the "UCF Faculty Clusters" text, and add `mr-2` to the `span` to remove the extra underline between the icon and text on hover.